### PR TITLE
Update dashboard FAQ

### PR DESCRIPTION
**Pull request by shhdgit**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added a new FAQ for TiDB Dashboard

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/docs/pull/5895
- Other reference link(s): [tidb-dashboard#902](https://github.com/pingcap/tidb-dashboard/issues/902)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -53,3 +53,24 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已在 v4.0.14/v5.0.3 及之后的版本修复，老版本可通过清空浏览器 Local Storage 解决。
+
+1. 打开 TiDB Dashboard 页面。
+
+2. 打开浏览器的开发者工具。各浏览器的打开方式不同。
+
+  - Firefox：菜单 ➤ Web 开发者 ➤ 切换工具箱（译者注：此处修改为最新的 Firefox Quantum），或者工具栏 ➤ 切换工具箱。 
+  - Chrome：菜单 ➤ 更多工具 ➤ 开发者工具。
+  - Safari：Develop ➤ Show Web Inspector。如果你看不到 Develop 菜单，点开 Preferences ➤ Advanced，然后点击 Show Develop menu in menu bar 复选框。
+  - Opera： Developer ➤ Web Inspector
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:68]:
```suggestion
```

File: dashboard/dashboard-faq.md [hunk 2]
```diff
@@ -53,3 +53,24 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。`prepared-plan-cache` 是实验性功能，在某些版本的 TiDB 中可能无法正常运行，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。可以通过修改 [TiDB 配置文件](/tidb-configuration-file.md#prepared-plan-cache)来关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已在 v4.0.14/v5.0.3 及之后的版本修复，老版本可通过清空浏览器 Local Storage 解决。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:59]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。较早版本可通过清空浏览器 Local Storage 解决。
```

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。`prepared-plan-cache` 是实验性功能，在某些版本的 TiDB 中可能无法正常运行，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。可以通过修改 [TiDB 配置文件](/tidb-configuration-file.md#prepared-plan-cache)来关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。较早版本可通过清空浏览器 Local Storage 解决。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:59]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你使用的是 v5.0.3 或 v4.0.14 之前的版本，可通过清空浏览器 Local Storage 解决。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: dashboard/dashboard-faq.md [hunk 4]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你使用的是 v5.0.3 或 v4.0.14 之前的版本，可通过清空浏览器 Local Storage 解决。
+
+1. 打开 TiDB Dashboard 页面。
+
+2. 打开浏览器的开发者工具。各浏览器的打开方式不同。
+
+  - Firefox：菜单 ➤ Web 开发者 ➤ 切换工具箱（译者注：此处修改为最新的 Firefox Quantum），或者工具栏 ➤ 切换工具箱。 
+  - Chrome：菜单 ➤ 更多工具 ➤ 开发者工具。
+  - Safari：Develop ➤ Show Web Inspector。如果你看不到 Develop 菜单，点开 Preferences ➤ Advanced，然后点击 Show Develop menu in menu bar 复选框。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:67]:
```suggestion
    - Firefox：菜单 ➤ Web 开发者 ➤ 切换工具箱（译者注：此处修改为最新的 Firefox Quantum），或者工具栏 ➤ 切换工具箱。 
    - Chrome：菜单 ➤ 更多工具 ➤ 开发者工具。
    - Safari：Develop ➤ Show Web Inspector。如果你看不到 Develop 菜单，点开 Preferences ➤ Advanced，然后点击 Show Develop menu in menu bar 复选框。
```

File: dashboard/dashboard-faq.md [hunk 5]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
+
+   以 Chrome 为例：
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:69]:
```suggestion
    以 Chrome 为例：
```

File: dashboard/dashboard-faq.md [hunk 6]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
+
+   ![打开开发者工具](/media/dashboard/dashboard-faq-devtools.png)
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:71]:
```suggestion
    ![打开开发者工具](/media/dashboard/dashboard-faq-devtools.png)
```

File: dashboard/dashboard-faq.md [hunk 7]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。`prepared-plan-cache` 是实验性功能，在某些版本的 TiDB 中可能无法正常运行，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。可以通过修改 [TiDB 配置文件](/tidb-configuration-file.md#prepared-plan-cache)来关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你使用的是 v5.0.3 或 v4.0.14 之前的版本，可通过清空浏览器 Local Storage 解决。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:59]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。
```

File: dashboard/dashboard-faq.md [hunk 8]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。`prepared-plan-cache` 是实验性功能，在某些版本的 TiDB 中可能无法正常运行，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。可以通过修改 [TiDB 配置文件](/tidb-configuration-file.md#prepared-plan-cache)来关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:59]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你的集群版本低于 v5.0.3 或 v4.0.14，可以根据以下步骤进行处理：
```

**Comment by shhdgit** [comment]
请问这个文档现在能合并吗，v4.0.14/5.0.3 已发布

File: dashboard/dashboard-faq.md [hunk 9]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。`prepared-plan-cache` 是实验性功能，在某些版本的 TiDB 中可能无法正常运行，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。可以通过修改 [TiDB 配置文件](/tidb-configuration-file.md#prepared-plan-cache)来关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你的集群版本低于 v5.0.3 或 v4.0.14，可以根据以下步骤进行处理：
+
+1. 打开 TiDB Dashboard 页面。
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:61]:
```suggestion
1. 打开 TiDB Dashboard 页面。 
```

File: dashboard/dashboard-faq.md [hunk 10]
```diff
@@ -53,3 +53,23 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:61]:
```suggestion
1. 打开 TiDB Dashboard 页面。
```

**Pull request merged**